cax/tasks/filesystem.py

```diff
@@ -145,15 +147,28 @@ def each_run(self):
             if 'host' not in data_doc or data_doc['host'] != config.get_hostname():
                 continue
                 
-            if self.node == data_doc['host'] and self.status == data_doc['status']:
+            if self.node == data_doc['host'] and self.status == data_doc['status'] and self.file == None and self.set == None:
+              #cax-status --disable_database_update --node tegner-login-1 --status verifying
               status_db = data_doc["status"]
               location_db = data_doc['location']
               logging.info("Ask for status %s at node %s: %s", self.node, status_db, location_db)
+            
+            if self.node == data_doc['host'] and self.status == data_doc['status'] and self.set != None and self.file == "all":
+              #example: cax-status --file all --disable_database_update --node tegner-login-1 --status verifying --set processed
+              if config.DATABASE_LOG is True:
+                self.collection.update({'_id' : self.run_doc['_id'],
+                                        'data': {'$elemMatch': data_doc}},
+                                       {'$set': {'data.$.status': self.set}})
+              logging.info('File/Data: %s is set from status %s to %s', data_doc['location'], data_doc['status'], self.set)
+            
+            if self.node == data_doc['host'] and self.status == data_doc['status'] and self.set != None and self.file == data_doc['location']:
+              #example: cax-status --file /cfs/klemming/projects/xenon/xenon1t/processed/pax_v4.10.0/160530_0505.root --disable_database_update --node tegner-login-1 --status verifying --set processed
```

Comment:
These are good tools to have, but I'd really avoid changing verifying to transferred manually.  This is supposed to be a checksum to make sure it actually worked...